ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -34,7 +34,7 @@
       "location" : "https://github.com/mosip/inji-openid4vp-ios-swift",
       "state" : {
         "branch" : "develop",
-        "revision" : "7529cd4b0342f8d5c98c37f7ed3961b7073bf730"
+        "revision" : "6473093e1494fe240b77f07a5fcb0164a8e70c55"
```

Comment:
why is there a change in this file? There is no ios side changes

Reply:
@Alka1703 We made changes on the openid4vp swift library as well so update the commit with the latest one